crates/cubecl-core/src/frontend/operation/branch.rs

```diff
@@ -15,11 +22,20 @@ pub fn select<C: CubePrimitive>(condition: bool, then: C, or_else: C) -> C {
     }
 }
 
+/// Same as [select] but with lines instead.
+#[allow(unused_variables)]
+pub fn select_many<C: CubePrimitive>(
```

Comment:
Funny name, why not select_line ?

Reply:
Or select element-wise

Reply:
Because a line as many elements. It doesn't select THE line, so select_line is wrong imo. I select elements broadcasted in the lines. Element-wise doesn't make sense either because everything is element-wise.